scripts/showinfo2.js

```diff
@@ -0,0 +1,11 @@
+function deploymenu2() {
+  const menubar = document.getElementById('menubar2');
+  menubar.style.display = 'flex';
+}
+document.getElementById('deploy-menu2').addEventListener('click', deploymenu2);
+
+function closemenu2() {
+  const menubar = document.getElementById('menubar2');
+  menubar.style.display = 'none';
+}
+document.getElementById('closebutton2').addEventListener('click', closemenu2);
```

Comment:
**[OPTIONAL]**

- Please, let's improve our naming convention to make our code more readable. As of right now, your function name and variable are not following the `camelCase` naming convention used in JavaScript, as this is considered bad practice. please let's try and improve this to make our code readable and maintainable.


e.g.. `deployMenu2`, `closeMenu2`, `menuBar`

Reply:
ok i already change into the camelCase